src/pyOpenMS/pxds/DocumentIdentifier.pxd

```diff
@@ -12,25 +12,25 @@ cdef extern from "<OpenMS/METADATA/DocumentIdentifier.h>" namespace "OpenMS":
         DocumentIdentifier(DocumentIdentifier) nogil except + # wrap-ignore
 
         # set document identifier (e.g. an LSID)
-        void setIdentifier(String id) nogil except +
+        void setIdentifier(String id) nogil except + # wrap-doc:Set document identifier (e.g. an LSID)
 
         # retrieve document identifier (e.g. an LSID)
-        String getIdentifier() nogil except +
+        String getIdentifier() nogil except + # wrap-doc:Retrieve document identifier (e.g. an LSID)
 
         # exchange content with @p from -> gets overwritten later
         # this does not work since some derived classes overwrite this (e.g. MSExperiment)
-        ## void swap(DocumentIdentifier from_) nogil except + 
+        ## void swap(DocumentIdentifier from_) nogil except +
 
         # set the file_type according to the type of the file loaded from (see FileHandler::Type) preferably done whilst loading
-        void setLoadedFileType(String file_name) nogil except +
+        void setLoadedFileType(String file_name) nogil except + # wrap-doc:Set the file_type according to the type of the file loaded from, preferably done whilst loading
 
         # get the file_type (e.g. featureXML, consensusXML, mzData, mzXML, mzML, ...) of the file loaded from
-        int getLoadedFileType() nogil except +
+        int getLoadedFileType() nogil except + # wrap-doc:Get the file_type (e.g. featureXML, consensusXML, mzData, mzXML, mzML, ...) of the file loaded from
```

Comment:
```suggestion
        int getLoadedFileType() nogil except + # wrap-doc:Get the file_type (e.g. featureXML, consensusXML, mzData, mzXML, mzML, ...) of the file loaded
```

Reply:
If the English used to describe something sounds somehow funky you can just change it, you do not need to copy 1:1 - just that you know :)

Not sure if mine is better :P

Reply:
Yes sure. 😀